docker/sandbox/flyte-entrypoint-default.sh

```diff
@@ -44,4 +44,7 @@ helm upgrade -n flyte --create-namespace flyte $charts --kubeconfig /etc/rancher
 
 wait-for-flyte.sh
 
+# With flytectl sandbox --source flag, we mount the root volume to user source dir that will create helm & k8s specific directory. It cause issue while doing fast register 
+rm -rf /root/.cache /root/.kube /root/.config
```

Comment:
And also the significance of these files and why removing them doesn't impact any functionality for the sandbox.

Reply:
no